Library/Homebrew/test/cask/system_command_spec.rb

```diff
@@ -1,4 +1,20 @@
 describe Hbc::SystemCommand, :cask do
+  describe "#initialize" do
+    describe "given some environment variables" do
+      subject do
+        described_class.new("/usr/bin/true",
+          env: { "A" => "1", "B" => "2", "C" => "3" })
+      end
+
+      it { is_expected.to be_an_instance_of described_class }
```

Comment:
Can you think of a less trivial expectation?

Reply:
Good point; I planned to do so initially. I ultimately fell back to the trivial expectation when I realized I had made the `env` accessor private (like all the other accessors are).

I wouldn’t recommend to make `env` public just for the sake of testing.
How about I simply remove this particular test?

Reply:
You could call `/usr/bin/env` instead of `true` and check if they are in fact set.

Reply:
@reitermarkus @claui Now we have a standard PATH I reckon it's always worth using `env` over `/usr/bin/env`

Reply:
@reitermarkus Glad you suggested that – it just revealed a major bug in my code!  
I wrote `sudo? ? env : {}`, while it obviously needs to be the other way around: `sudo? ? {} : env` 😊